cfme/ansible_tower/explorer.py

```diff
@@ -0,0 +1,144 @@
+from navmazing import NavigateToAttribute
+from widgetastic.widget import Text, View
+from widgetastic_patternfly import Dropdown
+from widgetastic.utils import VersionPick, Version
+from widgetastic_manageiq import Search, ItemsToolBarViewSelector, Button, Accordion, ManageIQTree
+
+from cfme.base.login import BaseLoggedInPage
+from cfme.utils.appliance import Navigatable
+from cfme.utils.appliance.implementations.ui import navigator, CFMENavigateStep
+
+
+class TowerExplorerAccordion(View):
+    @View.nested
+    class tower_explorer_providers(Accordion):  # noqa
+        ACCORDION_NAME = 'Providers'
+        tree = ManageIQTree()
+
+    @View.nested
+    class tower_explorer_systems(Accordion):  # noqa
+        ACCORDION_NAME = 'Configured Systems'
+        tree = ManageIQTree()
+
+    @View.nested
+    class tower_explorer_job_templates(Accordion):  # noqa
+        ACCORDION_NAME = 'Job Templates'
+        tree = ManageIQTree()
+
+
+class TowerExplorerProviderToolbar(View):
+    reload = Button(title=VersionPick({Version.lowest(): 'Reload current display',
```

Comment:
Please, do not use `VersionPick`. we already removed it from repo.

Reply:
Okay, what should I use instead?

Reply:
my mistook, ignore it. I forgot we are playing with `View`. WT `View` classes not supporting `contextual object` access. Yes, we can use `browser.appliance.version` mainly pointing to `appliance.version` in `methods` of `View` class but for `widget` initialization not possible with it. Same thing applicable to `ParametrizeView` :D